### ISSUE TITLE
Aria-label attribute of the "add selection" option is not defined which is appearing under "Command" combo edit text box

### ISSUE THREAD

**Issue by melode11** [SeleniumHQ/selenium-ide#1181]
## User Experience
Screen reader user will not able to understand the purpose of the control and will not able to access the control efficiently if name of the control is not defined
Note
User credentials should NOT be included in the bug.

## Repro Steps
Open Selenium IDE.
Navigate to  "Command" combo edit text box
Navigate on "add selection" option and inspect it
Verify the issue.
## Actual Result
 aria-label attribute of the "add selection" option is not defined which is appearing under "Command" combo edit text box due to which screen reader user will not able to understand the purpose of the control
## Expected Result
"aria-label=add selection" should be defined for  "add selection" option which is appearing under "Command" combo edit text box

## Note:
Same issue also exist with all options present under the "Command" combo edit
Same issue also exist with "Enable/Disabled command" button and "Add new window configuration" button[MAs4.1.2_Name of expand collapse command]
Same issue also exist with "Run current Testcase" button [MAS4.1.2_Aria-label]
Invalid aria-label attribute is defined for table rows[MAS4.1.2_Aria-label (2)]

## MAS Reference
MAS 4.1.2 - Name, Role, Value

## Environment

OS: Windows 10
Selenium IDE Version:  3.17.0
Browser: Chrome
Browser Version: 87.0.4280.141

**Comment by melode11** [comment]
PR merged, close issue.
